README.md

```diff
@@ -1,2 +1,26 @@
-# hello-react-front-end
-This exercise is going to have you create a React front-end with a Rails back-end and connect them to display a random message.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> The frontend for the Hello-world Webpage.
+> Link to [Backend app](https://github.com/Adanzeakonobi/hello-rails-back-end.git)
+## Built With
+- React
+- Redux
+- Axios
+- Gitflow
+
+
+👤 **Adaobi Vivian Okoye**
+
+- GitHub: [@githubhandle](https://github.com/adanzeakonobi) 
+- Twitter: [@twitterhandle](https://twitter.com/Adaebubemmuta)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/okoyeaadaobi)
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+	@@ -62,8 +55,6 @@ Feel free to check the [issues](../../issues/) page.
```

Comment:
- [ ] Please add the basic info about how to clone and run your app here in `Getting Started` section to make your README file more professional.